react/week5/githubusersprofile/src/Components/Profile.css

```diff
@@ -0,0 +1,18 @@
+div.profile {
```

Comment:
this file shouldn't be part of components... put Profile.js and Profile.css in a folder named Profile in Components folder

Reply:
yah, i get it,.Thanks